docs/cep/query-guide/functions/index.md

```diff
@@ -124,16 +124,16 @@ This stream application with the name `TestFunctions` creates a stream named `Fo
 On receiving the event, a query is executed that parses string data types using execution function (math) into required formats.
 After that, the data gets inserted into Kafka `SINK` using Apache Kafka IO function.
 
-```js
-@App:name('TestFunctions') 
+```sql
+    @App:name('TestFunctions') 
 
-CREATE SOURCE FooStream WITH (type='inMemory', topic='stock', map.type='json') (symbol string, price string, volume string);
+    CREATE SOURCE FooStream WITH (type='inMemory', topic='stock', map.type='json') (symbol string, price string, volume string);
```

Comment:
@StoyanTc, how can we rewrite this?

Reply:
The same here - `type='inMemory'` => `type='stream'`.